ExtraDry/ExtraDry.Core.Tests/Internals/FilterPropertyTests.cs

```diff
@@ -13,18 +12,19 @@ public void DefaultValues()
             var property = GetType().GetProperty(nameof(ValidProperty));
             var attribute = new FilterAttribute();
 
-            var filter = new FilterProperty(property!, attribute);
+            var filter = new FilterProperty(property!, attribute, property!.Name);
```

Comment:
I see that in one location you want to override the propertyName, but then in multiple locations pass the property name in in what appears to be a redundant fashion.  Can you make the third property optional so odd looking lines like this done occur?
e.g.
```
public FilterProperty(PropertyInfo property, AttributeInfo attribute, string? propertyName = null) {
    propertyName ??= property.Name;
...
}
```

Reply:
The third parameter has been optional and is no longer needed in this scenario.